Libraries/JavaScriptAppEngine/Initialization/InitializeJavaScriptAppEngine.js

```diff
@@ -154,6 +154,14 @@ function setUpGeolocation() {
   polyfillGlobal('geolocation', require('Geolocation'), GLOBAL.navigator);
 }
 
+function setUpProduct() {
+  let rnVersion = require('../../../package.json').version;
```

Comment:
@vjeux good points. I also think it's going to be a mess for the next while if libraries are trying to target specific RN versions until we hit 1.0 in 2017 or 2018 (maybe 2016, surprise me!)